NuRadioReco/examples/AraData/readARAexample.py

```diff
@@ -0,0 +1,19 @@
+import NuRadioReco.modules.io.araroot.readARAData
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('readARAexample')
+import matplotlib.pyplot as plt
+from NuRadioReco.utilities import units
+
+readARAData = NuRadioReco.modules.io.araroot.readARAData.readARAData()
+
+n=readARAData.begin("/home/uzair/Documents/AraRootStuff/event9129.root")
```

Comment:
It is not a good idea to have global paths on the repository. This will probably only work on Uzair's computer. It is better to let the user define the filename via the commandline: https://docs.python.org/3.3/library/argparse.html. If you want you can include a small example file in the repository and have the default point to it (with a relative path)

Reply:
I think when we did this, there was no 'draft' for pull requests yet. This is clearly not finished. 
Also, this won't work for anybody but Uzair, because you need a working version of ARAroot ;)

Reply:
But in principle, you are correct!

Reply:
Okay, I just thought it was done and Uzair was waiting for someone to approve :-)

Reply:
Thanks Christoph! For pointing that out. I am aware of that. Will certainly do that. Right now I am trying to get the interpolator running AraFile reader. In ROOT I am required to include a header file for it. In Python I am trying to call it through shared library .so file. Have been unsuccessful at it till now. Hopefully something will work out this week.

Reply:
@uzairlatif90: don't try to think ROOT so much. Scipy has an incredible wealth of interpolation methods. https://docs.scipy.org/doc/scipy/reference/interpolate.html
It might be much easier -- and better for the sustainability -- to use a python for the interpolation. We are happy to help, when it comes to these programming things.